tox.ini

```diff
@@ -2,7 +2,7 @@
 envlist = py{27,35}
 skipsdist = true
 
-[pep8]
+[pycodestyle]
```

Comment:
Cool, looks like they just released the fix for this 2 weeks ago.  I tried this right after the pep8 -> pycodestyle rename, but the code didn't support it yet.